dist/index.html

```diff
@@ -3,17 +3,19 @@
     <head>
         <meta charset="UTF-8" />
         <title>Voter-Science Pin Color</title>
+
+        <link rel="stylesheet" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css">
```

Comment:
Is there a way to avoid css-specifics within the plugins?    
If each plugin has its own css, then they can diverge and start looking different. 
We have a CDN and could host a common global CSS  file; or pull this from trc-react?

Reply:
We should merge and host the 2 global CSS files on the CDN. Could you provide the URL so that I can swap them?

They are just CSS resets with some simple common rules. They just rarely, if ever, change.

Reply:
I added the new CSS file and removed the 2 old ones. 👍